misc/inmanta-workon-register.sh

```diff
@@ -170,8 +177,11 @@ function __inmanta_workon_activate {
     declare OLD_PS1="$PS1"
     source "$activate"
     export PS1="($env_name) $OLD_PS1"
+    export INMANTA_CONFIG_ENVIRONMENT=$env_id
```

Comment:
We should probably unset this one on deactivate. For `PS1` we don't do it because `virtualenv` does it for us.

Reply:
(See the `activate` script in the `bin` dir of any venv you've got lying around.)